documentation/assemblies/metrics/assembly-metrics.adoc

```diff
@@ -3,34 +3,38 @@
 // deploying/deploying.adoc
 
 [id='assembly-metrics-{context}']
-= Introducing Metrics to Kafka
+= Introducing metrics to Kafka
 
-This section describes how to monitor your Strimzi deployment.
+[role="_abstract"]
+You can use Prometheus and Grafana to monitor your Strimzi deployment.
 
 Depending on your requirements, you can:
 
-* xref:assembly-metrics-setup-{context}[Set up and deploy Prometheus and Grafana]
-* xref:assembly-kafka-exporter-{context}[Configure the `Kafka` resource to deploy Kafka Exporter with your Kafka cluster]
-+
-Kafka Exporter provides additional monitoring related to consumer lag.
+* xref:assembly-metrics-setup-{context}[Set up and deploy Prometheus to expose metrics]
+* xref:proc-metrics-grafana-dashboard-{context}[Use Grafana to present the Prometheus metrics]
+* xref:proc-metrics-kafka-deploy-options-{context}[Deploy Kafka Exporter to provide additional metrics]
 
 With Prometheus and Grafana set up, you can use the example Grafana dashboards provided by Strimzi for monitoring.
 
-Additionally, you can configure your deployment to track messages end-to-end by link:{BookURLUsing}#assembly-distributed-tracing-str[setting up distributed tracing^], as described in the _Using Strimzi_ guide.
+Additionally, you can configure your deployment to track messages end-to-end by link:{BookURLUsing}#assembly-distributed-tracing-str[setting up distributed tracing^].
 
+NOTE: The resources referenced in this section are intended as a starting point for setting up monitoring, but they are provided as examples only.
+If you require further support on configuring and running Prometheus or Grafana in production, try reaching out to their respective communities.
```

Comment:
Can we be more specific here that this applies to the Prometheus / Grafana installation files? Nor necessarily to the Grafana dashboards for example?

Reply:
I changed to this:

> Strimzi provides example installation files for Prometheus and Grafana. You can use these files as a starting point when trying out monitoring of Strimzi. For further support, try engaging with the Prometheus and Grafana developer communities.

Reply:
That sounds good.